src/ColumnsSummary.php

```diff
@@ -94,6 +101,17 @@ public function add(Row $row)
 	 */
 	public function render($key)
 	{
+		/**
+		 * Renderer function may be used
+		 */
+		try {
+			return $this->useRenderer($key);
```

Comment:
Wouldn't it be a little bit more readable not to catch an exception but checking for instance `$this->renderer instanceof Renderer ` instead?